.github/workflows/newman.yml

```diff
@@ -409,11 +369,11 @@ jobs:
           done
 
       - name: Change test execution status
-        if: ${{ contains(fromJson('["testing", "unstable"]'), inputs.stability) }}
+        if: ${{ contains(fromJson('["testing","unstable"]'), inputs.stability) || failure() }}
```

Comment:
test execution status must be updated only on failure ?
if yes, I guess it should be `&&` instead of `||`

Reply:
in fact the status change must be performed in the case of success or failure, the problem I've seen before is that when tests fail this step is bypassed whereas even in the case of fail you must change the status to fail